src/App.tsx

```diff
@@ -1,18 +1,102 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useSearchParams } from 'react-router-dom';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+
+const items: { title: string, id: number }[] = getNumbers(1, 42)
```

Comment:
You could create a separate interface for the object

Reply:
Done